src/App.tsx

```diff
@@ -49,7 +49,7 @@ const App: FC = () => {
               <Switch>
                 <Route exact path="/" component={HomePageRedirect} />
                 <Route path="/:lng?/newsflash/:id" component={HomePage} />
-                <Route path="/popup-redirect" component={PopUpRedirect}></Route>
+                <Route path="/login-popup-redirect" component={PopUpRedirect} />
```

Comment:
better moving all routes to a constant file

Reply:
maybe it will be best to open a seperatd issue for that? @yuvalbl

Reply:
indeed, open an issue for it please